storage/storage.go

```diff
@@ -22,15 +23,13 @@ type S3BucketStorage struct {
 
 // NewS3Bucket creates a new S3 bucket storage
 func NewS3Bucket(config *config.Config) Storage {
-	sess, err := session.NewSessionWithOptions(session.Options{
-		Config: aws.Config{
-
-			Region:      aws.String(config.AWSRegion),
-			Credentials: credentials.NewEnvCredentials(),
-			Endpoint:    aws.String(config.S3Endpoint),
-			DisableSSL:  aws.Bool(config.S3DisableSSL),
-		},
+	sess, err := session.NewSession(&aws.Config{
+		Endpoint:   aws.String(config.S3Endpoint),
+		DisableSSL: aws.Bool(config.S3DisableSSL),
 	})
+	if err != nil {
+		log.Println(err)
```

Comment:
I kind of think this should be a panic rather than just a log message, because if these sessions fail, it's going to have a bad time later on.  Same with the other 3 locations.

Reply:
Fair enough - I wanted the logs while I was debugging the connections but will change it to be a panic